pandas-stubs/plotting/_misc.pyi

```diff
@@ -1,85 +1,82 @@
 from typing import (
     Any,
+    Hashable,
+    Literal,
     Sequence,
+    Union,
 )
 
-from matplotlib.axes import Axes as PlotAxes
+from matplotlib.axes import Axes
+from matplotlib.colors import Colormap
 from matplotlib.figure import Figure
+from matplotlib.table import Table
 import numpy as np
 from pandas.core.frame import DataFrame
 from pandas.core.series import Series
 
+from pandas._typing import HashableT
+
+_Color = Union[str, Sequence[float]]
+
 def table(
-    ax,
-    data,
-    rowLabels=...,
-    colLabels=...,
-): ...
+    ax: Axes,
+    data: DataFrame | Series,
+    **kwargs,
+) -> Table: ...
 def register() -> None: ...
 def deregister() -> None: ...
```

Comment:
I don't see `register()` and `deregister()` as documented, probably can be deleted.

Reply:
These are short name implementations of `(de)register_matplotlib_converters`.  Keeping close to actual implementation to avoid confusion.  These are exported and then used in the `__init__`